account_credit_control/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Account Credit Control',
-    'version': '9.0.0.1.0',
+    'version': '9.0.0.2.0',
```

Comment:
Version is incorrect (well, it was also incorrect on first iteration), but here you don't get anything changing it.